### ISSUE TITLE
Panic on update

### ISSUE THREAD

**Issue by twoseat** [vmware-archive/pcfdev#145]
While upgrading from 0.19 to 0.20 using cf cli 6.22.1 I found the following issue:

`cf dev stop` (nothing running, unsurprising as I'd just rebooted)

`./pcfdev-v0.20.0+PCF1.8.0-osx` 
`Plugin successfully upgraded. Current version: 0.20.0. For more info run: cf dev help`

`cf dev start`

``` FAILED
Error: old version of PCF Dev already running, please run `cf dev destroy` to continue.
panic: This shouldn't print anything

goroutine 1 [running]:
panic(0x4402a00, 0xc4201d00c0)
    /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/pivotal-cf/pcfdev-cli/vendor/github.com/cloudfoundry/cli/cf/terminal.(*terminalUI).PanicQuietly(0xc420182ec0)
    /ext-go/1/src/github.com/pivotal-cf/pcfdev-cli/vendor/github.com/cloudfoundry/cli/cf/terminal/ui.go:180 +0x6d
github.com/pivotal-cf/pcfdev-cli/vendor/github.com/cloudfoundry/cli/cf/terminal.(*terminalUI).Failed(0xc420182ec0, 0xc4201d4180, 0x57, 0x0, 0x0, 0x0)
    /ext-go/1/src/github.com/pivotal-cf/pcfdev-cli/vendor/github.com/cloudfoundry/cli/cf/terminal/ui.go:176 +0x227
github.com/pivotal-cf/pcfdev-cli/plugin.(*NonTranslatingUI).Failed(0xc42017dd60, 0xc4201d4120, 0x57, 0x0, 0x0, 0x0)
    <autogenerated>:6 +0x7b
github.com/pivotal-cf/pcfdev-cli/plugin.(*Plugin).Run(0xc42017b170, 0x46c8ee0, 0xc42017dd80, 0xc42000c260, 0x2, 0x2)
    /ext-go/1/src/github.com/pivotal-cf/pcfdev-cli/plugin/plugin.go:56 +0x1e4
github.com/pivotal-cf/pcfdev-cli/vendor/github.com/cloudfoundry/cli/plugin.Start(0x46c1b60, 0xc42017b170)
    /ext-go/1/src/github.com/pivotal-cf/pcfdev-cli/vendor/github.com/cloudfoundry/cli/plugin/plugin_shim.go:30 +0x288
main.main()
    /ext-go/1/src/github.com/pivotal-cf/pcfdev-cli/main.go:113 +0xb2f
```

Destroying and then starting appears to work fine, but as the output notes, the panic shouldn't appear.

**Comment by sclevine** [comment]
Thanks for the report! We think this has to do with changes in the most recent cf CLI. The panic is harmless, and it will likely go away if you downgrade your cf CLI. We will address this in the next PCF Dev release.

**Comment by sclevine** [comment]
Fixed in 0.21.0.
